FlightJournal.Web/Startup.cs

```diff
@@ -15,7 +15,8 @@ public void Configuration(IAppBuilder app)
             System.Data.SqlClient.SqlAuthenticationProvider.SetProvider(System.Data.SqlClient.SqlAuthenticationMethod.ActiveDirectoryInteractive, new SqlAppAuthenticationProvider());
 
             ConfigureAuth(app);
-            ConfigureOpenGliderNetworkFlightLogMonitor(app);
+//            ConfigureOpenGliderNetworkFlightLogMonitor(app); // think this is obsolete ?
```

Comment:
ConfigureOpenGliderNetworkFlightLogMonitor is on live used by one paying club... 
But i agree we should deprecate once we are confident on this code.